src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs

```diff
@@ -512,8 +512,9 @@ private void OnMouseLeftButtonDown(object sender, MouseButtonEventArgs e)
 
             if (!ViewModel.IsDragging) return;
 
-            var nodesToHidePreview = this.ChildrenOfType<NodeView>()
-                .Where(view => !view.PreviewControl.IsHidden && !view.PreviewControl.StaysOpen);
+            var nodesToHidePreview = this.ChildrenOfType<NodeView>().Where(view =>
+                view.HasPreviewControl && !view.PreviewControl.IsHidden && !view.PreviewControl.StaysOpen);
```

Comment:
it might be good to add a comment for why this extra check was needed, but not necessary.